AmazonChimeSDKDemo/AmazonChimeSDKDemo/CaptionsModel.swift

```diff
@@ -8,6 +8,7 @@
 
 import AmazonChimeSDK
 import UIKit
+import CoreMIDI
```

Comment:
Is this needed? I don't see any `MIDI` related changes here.

Reply:
will remove in my next commit.